app/javascript/mastodon/components/media_gallery.jsx

```diff
@@ -57,7 +58,7 @@ class Item extends PureComponent {
 
   hoverToPlay () {
     const { attachment } = this.props;
-    return !this.getAutoPlay() && attachment.get('type') === 'gifv';
+    return !this.getAutoPlay() && ['gifv', 'video'].includes(attachment.get('type'));
```

Comment:
Maybe the `['gifv', 'video'].includes(…)` part should be a helper function, as its repeated in many places

Reply:
This component needs to be refactored into TypeScript, then this won't be needed at all.